### PR TITLE
fix(Testimonial): Fixes visual regression + renames to TestimonialDeprecated

### PR DESCRIPTION
**Pull request by cstatro**
### PR Checklist
This PR kicks off the deprecation process for our testimonial component ( our new one will be leveraging floating card) and fixes the overlap visual bug caused by the Avatar component . 

before:

image

after:
image

- [X] Related to JIRA ticket:  GROW-2490
- [X] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.6-alpha.1614ee.0<br/>@codecademy/gamut-labs@19.16.18-alpha.1614ee.0<br/>@codecademy/styleguide@42.3.8-alpha.1614ee.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61084ec0415ce0d681626f0c--gamut-preview.netlify.app>https://61084ec0415ce0d681626f0c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61084ec0415ce0d681626f0c>Deploy Logs</a></p>

File: packages/gamut-labs/src/TestimonialDeprecated/index.tsx [hunk 1]
```diff
@@ -126,14 +131,13 @@ export const Testimonial: React.FC<TestimonialProps> = ({
...
-            flexDirection="row"
             flexWrap="wrap"
             alignItems="start"
             textColor={mode === 'dark' ? 'beige' : 'navy'}
           >
-            <FlexBox flexWrap="wrap" flexGrow={1} flexBasis="0">
+            <FlexBox flexDirection="column">
               {imageUrl && (
-                <Box width="77px" height="77px" mb={16} mr={16}>
+                <Box width="77px" height="77px" mb={48} mr={16}>
```
Comment (cstatro) [inline packages/gamut-labs/src/TestimonialDeprecated/index.tsx:140]: this fixes the issue we're seeing on our current implementation of testimonial, but doesn't really address the issue of the `Avatar` component having some visual side effects on nearby elements. That will require either: 
a. we build a new `Avatar` component
b. we iterate on the existing one to make sure that avatars have a container wrapping them that prevents overlapping.

**Review by codecaaron — APPROVED** [review]
Let's merge this with Next release so we can get update the names at once - since this is technically a breaking change.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.6-alpha.216478.0<br/>@codecademy/gamut-labs@19.16.18-alpha.216478.0<br/>@codecademy/styleguide@42.3.8-alpha.216478.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61085cabe990ee2dd47a580a--gamut-preview.netlify.app>https://61085cabe990ee2dd47a580a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61085cabe990ee2dd47a580a>Deploy Logs</a></p>

**Review by JoshuaKGoldberg — APPROVED** [review]
🦖 ☄️

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.10-alpha.1558a3.0<br/>@codecademy/gamut-labs@19.18.2-alpha.1558a3.0<br/>@codecademy/styleguide@42.6.2-alpha.1558a3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6108957242afb7b496a2531b--gamut-preview.netlify.app>https://6108957242afb7b496a2531b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6108957242afb7b496a2531b>Deploy Logs</a></p>

**Pull request merged**